### PR TITLE
Managing security with.. translation DONE

### PR DESCRIPTION
**Pull request by PiperLiu**
Managing security with Azure Lighthouse and Azure Arc is translated, NEEDING CHECK.

### CODE REVIEW

**Review by iamanother — COMMENTED** [review]
Good job!

File: content/cloud-advocate/2020-01/Managing-security-with-Azure-Lighthouse-and-Azure-Arc/index.md [hunk 1]
```diff
@@ -1,67 +1,78 @@
...
 # Managing security with Azure Lighthouse and Azure Arc
 
+# 基于 Azure Lighthouse 和 Azure Arc 的安全管理
+
 <ContentMeta />
 
 I've previously blogged about Azure Lighthouse, for [managing multiple difference Azure tenancies](https://techcommunity.microsoft.com/t5/ITOps-Talk-Blog/Manage-multiple-Azure-tenancies-with-Azure-Lighthouse/ba-p/833928?WT.mc_id=itopstalk-blog-socuff). This capability is useful for both Managed Service Providers, with support arrangements for multiple customers, and for large or complex Enterprise organizations (for example, if there are sub-brands or franchisees). Onboarding is done via deploying an Azure Resource Manager template or publishing an Azure Marketplace offer (public or private), which enables delegated administration into the other tenancies.
 
- 
+我以前写过一篇关于 Azure Lighthouse 的博文，介绍了[管理不同的 Azure 租户](https://techcommunity.microsoft.com/t5/ITOps-Talk-Blog/Manage-multiple-Azure-tenancies-with-Azure-Lighthouse/ba-p/833928?WT.mc_id=itopstalk-blog-socuff)。这个功能对于面向多客户的托管服务提供商，以及庞大复杂的企业组织（比如，有子品牌的或连锁加盟的企业集团）来讲，都很有用。通过部署 Azure 资源管理器模板，或发布一个（公有或私有的） Azure 市场报价，即可以完成交付，这将把管理权委托给其他租户。
```
Comment (iamanother) [inline content/cloud-advocate/2020-01/Managing-security-with-Azure-Lighthouse-and-Azure-Arc/index.md]: 管理多个不同的Azure租户

**Review by iamanother — APPROVED** [review]

File: content/cloud-advocate/2020-01/Managing-security-with-Azure-Lighthouse-and-Azure-Arc/index.md (hunk 1 above)
Comment (PiperLiu) [reply content/cloud-advocate/2020-01/Managing-security-with-Azure-Lighthouse-and-Azure-Arc/index.md]: Thanks, already adopted your idea. Committed done.

Comment (PiperLiu) [reply content/cloud-advocate/2020-01/Managing-security-with-Azure-Lighthouse-and-Azure-Arc/index.md]: > 管理多个不同的Azure租户

Thanks. Already adopted your idea, committed done.

**Pull request merged**